src/index.js

```diff
@@ -9,6 +9,7 @@ const manager = new TasksManager();
 
 const updateTask = (event, index, focus = false) => {
   if (event.key === 'Enter' || focus) {
+    // console.log("test");
```

Comment:
> * [ ] Please remove commented code as its regard a bad practice because it makes your code not clean. 
For a refresher on the JavaScript best practises, please check this  [link](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)